source-gallery/src/main/java/com/google/android/apps/muzei/gallery/GalleryImportPhotosDialogFragment.java

```diff
@@ -0,0 +1,100 @@
+package com.google.android.apps.muzei.gallery;
+
+import android.app.Dialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.pm.ActivityInfo;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.DialogFragment;
+import android.support.v7.app.AlertDialog;
+import android.widget.ArrayAdapter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GalleryImportPhotosDialogFragment extends DialogFragment {
+    public static final String TAG = GalleryImportPhotosDialogFragment.class.getSimpleName();
```

Comment:
Use "GalleryImportPhotos" as ProGuard will rename the class itself

Reply:
Done